README.md

```diff
@@ -54,7 +54,12 @@
 ### Key Features <a name="key-features"></a>
 
```

Comment:
- [ ] In the Acknowledgments section of the README file, Please make sure that you give appropriate credit to the author [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) as requested in the project requirements.

![image](https://github.com/martinkarugaba/budget_app/assets/83803414/4cf95aa9-15ef-4379-ad08-e940332c0eba)